client/pages/Settings.tsx

```diff
@@ -97,47 +97,47 @@ export default function Settings() {
 
     useEffect(() => {
         API.get("/users/me")
-        .then((data: User) => {
-            setUser(data);
-
-            if (data.isAdmin) {
-                API.get("/users")
-                .then((users: string[]) => {
-                    for (let u of users) {
-                        if (u === data.username) continue; // skip self
-
-                        API.get(`/users/${u}/details`)
-                        .then((user) => {
-                            setAllUsers(prevAllUsers => {
-                                return [...prevAllUsers, user];
-                            });
+            .then((data: User) => {
+                setUser(data);
+
+                if (data.isAdmin) {
+                    API.get("/users")
+                        .then((users: string[]) => {
+                            for (let u of users) {
+                                if (u === data.username) continue; // skip self
+
+                                API.get(`/users/${u}/details`)
+                                    .then((user) => {
+                                        setAllUsers(prevAllUsers => {
+                                            return [...prevAllUsers, user];
+                                        });
+                                    });
+                            }
                         });
-                    }
-                });
-            }
-        });
+                }
+            });
     }, []);
 
     const handleImportSubmit = (event) => {
         event.preventDefault();
         const formData = new FormData(event.currentTarget);
 
-        for(const pair of formData.entries()) {
+        for (const pair of formData.entries()) {
             const file = pair[1];
 
-            if(file instanceof Blob && file.size > 0) {
+            if (file instanceof Blob && file.size > 0) {
                 var sendFormData = new FormData();
                 sendFormData.append('file', file);
-                API.post(`/importing?csv_type=${pair[0]}`,sendFormData)
-                .then(() => navigate("/"));
+                API.post(`/importing?csv_type=${pair[0]}`, sendFormData)
+                    .then(() => navigate("/"));
             }
         }
     }
 
     const handleExportClick = (exportType: string) => {
-        if(exportType === "csv") {
+        if (exportType === "csv") {
```

Comment:
For the sake of consistency with the rest of the code base, there shouldn't be spaces after `if`, `for`, ...

The same goes for other parts of the code in this PR where a space was added after such keywords